srcCpp/perftest_publisher.cxx

```diff
@@ -1544,15 +1531,24 @@ class LatencyListener : public IMessagingCB
             default:
                 break;
             }
-
-            printf("Serialization/Deserialization: %0.3f us / %0.3f us / TOTAL: "
-                    "%0.3f us\n",
-                    serializeTime,
-                    deserializeTime,
-                    serializeTime + deserializeTime);
+            // _printer->print_serialization(serializeTime, deserializeTime);
         }
       #endif
 
+
+        _printer->print_latency_summary(
+                totalSampleSize,
+                latency_ave,
+                latency_std,
+                latency_min,
+                latency_max,
+                _latency_history,
+                count,
+                serializeTime,
+                deserializeTime,
+                outputCpu);
+        fflush(stdout);
```

Comment:
the fflush you can add it inside the print print_latency_summary. Do the same for throughput one.

Reply:
ok